src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmployeeController.php

```diff
@@ -476,6 +477,10 @@ protected function getEmployeeFormHandler()
     protected function getErrorMessages(Exception $e)
     {
         return [
+            UploadedImageConstraintException::class => $this->trans(
+                'The file extension for the avatar is not supported. Valid ones are: gif, jpg, jpeg, jpe, png, webp.',
```

Comment:
jpe? does it exist 😄 ?

Reply:
It does ! https://fileinfo.com/extension/jpe, and, according to the original message, it is accepted cf issue description https://github.com/PrestaShop/PrestaShop/issues/28985